docs/pricing.rst

```diff
@@ -0,0 +1,124 @@
+
+Concepts
+--------
+
+Events
+~~~~~~
+
+An event is a crash report (also known as exception or error) that is sent to Sentry’s server to be processes. Additional metadata around an event (breadcrumbs, user context, environment, etc) do not count as separate events.
+
+Capacity
+~~~~~~~~
+
+Capacity is the total number of events sent by and billed to your organization. Capacity comes in two types: on-demand and reserved.
+
+On-demand Capacity
+``````````````````
+
+On-demand capacity is billed at the end of each billing cycle. You are charged per processed event, rounded up to the nearest cent.
+
+
+Reserved Capacity
+`````````````````
+
+Reserved capacity allows you to prepay annually for significant discounts (up to 75%). Your reserved capacity is always applied before on-demand capacity. Unused reserved capacity expires at the end of each billing month.
+
+Free Capacity
+`````````````
+
+All Sentry organizations receive 10,000 events per month of free capacity. You retain your free capacity when switching to a paid plan.
+
+Billing Cycle
+~~~~~~~~~~~~~
+
+By default, your account will be billed monthly, starting on the date you switch to a paid account. If you purchase reserve capacity, which requires annual billing, the annual billing cycle start date will match the current monthly billing start date. Any unused capacity will be discarded at the end of each monthly billing cycle. The minimum charge for any billing month is $1.
+
+Billing Controls
+~~~~~~~~~~~~~~~~
+
+On-demand spending cap
+``````````````````````
+
+You can set a maximum monthly on-demand bill amount by setting an on-demand spending cap. Once you reach the spending cap, you will still be able to access Sentry and prior event data. However, any additional events you send will be rejected.
+
+Per-minute rate limits
+``````````````````````
+
+You can set the maximum number of events per minute that Sentry will accept. This allows you to protect yourself from consuming all of your month’s capacity in the event of sudden spikes.
+
+Plans
+-----
+
+Paid
+~~~~
+
+Every paid plan includes 50,000 reserved events per month. Additional capacity can be purchased as on-demand or reserved. Reserved capacity can be upgrade mid-month and consumed capacity will be recalculated based on the new reserved capacity level.
+
+Trial
+`````
+
+The paid plan comes with a 14-day trial of Sentry with unlimited events and team members. Trials have access to all features and email support. Trials have a default rate limit of 500 events/minute. If you need more or an unlimited rate limit during trial, please contact us sales@sentry.io.
+
+Free
+~~~~
+
+Sentry’s free tier provides Sentry’s core functionality for hobbyist pursuing side projects. It includes 10,000 events per month, 1 team member, and community support.
```

Comment:
> Sentry’s free tier provides Sentry’s core functionality

It might be good to be more specific either about what is included or what is excluded.